pkg/build/package.go

```diff
@@ -97,6 +97,7 @@ pkgname = {{.PackageName}}
 pkgver = {{.Origin.Version}}-r{{.Origin.Epoch}}
 arch = {{.Arch}}
 size = {{.InstalledSize}}
+origin = {{.PackageName}}
```

Comment:
This should be `.Origin.Name` to match `abuild`:

```suggestion
origin = {{.Origin.Name}}
```